README.md

```diff
@@ -161,6 +161,8 @@ ReactRailsUJS.useContext(myCustomContext)
 
 If `require` fails to find your component, [`ReactRailsUJS`](#ujs) falls back to the global namespace, described in [Use with Asset Pipeline](#use-with-asset-pipeline).
 
+In some cases, having multiple `require.context` may be desired. If so, an array of `require.context` can be passed to `ReactRailsUJS.useContexts`. With an array of contexts, an attempt will be made to `require` the component from each context before falling back to the global namespace as described above.
```

Comment:
@justin808 Had a go at adding a specific use case for multiple contexts, what do you think? @cymen not sure if there's a different example you had in mind.

```suggestion
In some cases, having multiple `require.context` may be desired - for example, if you have additional entry points to create JavaScript files for different routes, you would add multiple `require.context` to your `server_rendering.js` to allow for [Server-Side Rendering](#server-side-rendering) across your application. If so, an array of `require.context` can be passed to `ReactRailsUJS.useContexts`. With an array of contexts, an attempt will be made to `require` the component from each context before falling back to the global namespace as described above.

```javascript
// server_rendering.js
var componentsRequireContext = require.context('components', true);
var searchRequireContext = require.context('search', true);
var checkoutRequireContext = require.context('checkout', true);

var ReactRailsUJS = require('react_ujs');
ReactRailsUJS.useContexts([
  componentsRequireContext,
  searchRequireContext,
  checkoutRequireContext
]);
```
```

Reply:
Updated the suggestion based on changes from my other PR to hopefully provide a clearer usage example.

Reply:
This looks good to me (as mentioned elsewhere). Thanks for fleshing it out. I added another example use case with c5cd6c6c627f73ecb6fe38a2a0d989ed428e310d which is the one that sent me down this path of wanting multiple contexts (we've been using a forked version of react-rails in production for quite a while and it's been working well).

Reply:
Really interesting to hear your use case for this.